woocommerce/woocommerce
# CYS - E2E tests: Add footer section e2e tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Part of https://github.com/woocommerce/woocommerce/issues/45344



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Run `pnpm run --filter="@woocommerce/plugin-woocommerce" test:e2e-pw plugins/woocommerce/tests/e2e-pw/tests/customize-store/assembler/footer.spec.js` and make sure they pass.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
CYS - E2E tests: Add footer section E2E tests
#### Comment 

</details>
